addons/weapons/CfgWeapons.hpp

```diff
@@ -463,13 +463,26 @@ class CfgWeapons {
         delete OpticsModes;
         delete WeaponSlotsInfo;
         class ItemInfo : CBA_MiscItem_ItemInfo {
-            mass = 64;
+            mass = 141;
             allowedSlots[] = { 901 };
         };
     };
     class launch_B_Titan_short_F;
     class UK3CB_BAF_Javelin_Launcher : launch_B_Titan_short_F {
         modes[] = { "Single", "TopDown" };
+        class WeaponSlotsInfo {
```

Comment:
I think this (and the other two below) should inherit their parent's `WeaponSlotsInfo` class.
This change would wipe `allowedSlots[] = {};` from this `WeaponSlotsInfo` class. Don't know if that would have any negative effects, but it's best practise to always inherit from the parent if the class already does so, or if the definition already exists somewhere in the inheritance tree.
e.g `class UK3CB_BAF_Javelin_Launcher: launch_B_Titan_short_F {` already contains a `class WeaponSlotsInfo: WeaponSlotsInfo {` definition, so that inheritance should be maintained.

Reply:
Resolved by https://github.com/uksf/modpack/pull/620/commits/db4d664698ee57e4b15ec2e90f0def8de557dff5

Reply:
Resolved by https://github.com/uksf/modpack/pull/620/commits/db4d664698ee57e4b15ec2e90f0def8de557dff5